package.json

```diff
@@ -26,12 +26,11 @@
   ],
   "homepage": "https://github.com/amilajack/eslint-plugin-compat#readme",
   "scripts": {
-    "build": "cross-env NODE_ENV=production rm -rf lib && babel src --out-dir lib --source-maps inline",
-    "flow": "flow",
-    "flow-typed": "flow-typed install --ignoreDeps peer dev",
-    "lint": "eslint --ignore-path .gitignore .",
+    "build": "cross-env NODE_ENV=production rm -rf lib && babel src --out-dir lib --source-maps inline --extensions '.ts'",
+    "lint": "eslint --ignore-path .gitignore --ext .js,.ts .",
     "spec": "jest",
     "test": "yarn lint && yarn build && yarn spec",
+    "tsc": "tsc",
```

Comment:
We could remove this line and calling `yarn tsc` would still work exactly the same. Its a pattern i used in other projects but I'm seeing it as being verbose now. Thoughts on this?

Reply:
it'd be needed for `npm run tsc`, which i'd suggest switching to anyways :-)

Reply:
But we expect all contributors to be using `yarn` so I don't think this would give us much value.

Reply:
I'm interested in considering the tradeoffs in this but might be better to discuss this in a new issue.

Reply:
is yarn no longer good?